README.md

```diff
@@ -917,6 +917,62 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='comments'></a>(<a href='#comments'>link</a>) **Comment blocks should use single-line comments (`//` for code comments and `///` for documentation comments)**, rather than multi-line comments (`/* ... */` and `/** ... */`). [![SwiftFormat: blockComments](https://img.shields.io/badge/SwiftFormat-blockComments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#blockComments)
+
+  <details>
+
+  ```swift
+  // WRONG
+
+  /**
+  * A planet that exists somewhere in the universe.
+  *
+  * Planets have many properties. For example, the best planets
+  * have atmospheres and bodies of water to support life.
+  */
+  class Planet {
+    /**
+      Terraforms the planet, by adding an atmosphere and ocean
+      that is hospitable for life.
+      - Precondition: the planet does not already have an atmosphere.
```

Comment:
Do you think we should add a precondition to the code? We could assume it's inside of `generateAtmosphere()` though I think it would be most logical for it to live in `terraform()`.

Reply:
The "precondition" itself is a bit tangential to the purpose of the sample code, maybe we should simplify the comment instead to not imply the presence of an actual `precondition(...)`. Maybe something like `- Note:` would work here instead. Thoughts?

Reply:
Yes I like the note idea.